### PR TITLE
[MAINT] Update to keep up with poldracklab/niworkflows#299

### PR DESCRIPTION
**Pull request by oesteban**
And poldracklab/smriprep#51 as well.

### CODE REVIEW

File: fmriprep/cli/run.py [hunk 1]
```diff
@@ -606,8 +607,9 @@ def build_workflow(opts, retval):
 
     # First check that bids_dir looks like a BIDS folder
     bids_dir = os.path.abspath(opts.bids_dir)
+    layout = BIDSLayout(bids_dir, validate=False)
```
Comment (effigies) [inline fmriprep/cli/run.py]: Just out of curiosity, why are we using `validate=False`?

File: fmriprep/workflows/fieldmap/phdiff.py [hunk 2]
```diff
@@ -73,7 +73,8 @@ def _pick1st(inlist):
         return inlist[0]
 
     # Read phasediff echo times
-    meta = pe.Node(ReadSidecarJSON(), name='meta', mem_gb=0.01, run_without_submitting=True)
+    meta = pe.Node(ReadSidecarJSON(bids_validate=False), name='meta',
+                   mem_gb=0.01, run_without_submitting=True)
```
Comment (effigies) [inline fmriprep/workflows/fieldmap/phdiff.py]: For nodes that construct BIDS layouts, we may want to consider whether to use `run_without_submitting`. Large datasets can take a long time, and `run_without_submitting` means we block the master thread.

Maybe we can time the first creation, and if it's >2sec (or something) go ahead and submit?

File: fmriprep/cli/run.py (hunk 1 above)
Comment (oesteban) [reply fmriprep/cli/run.py]: Assuming that `--skip-bids-validator` was not set. We could actually tail it to that and force going through some validation.

Comment (oesteban) [reply fmriprep/cli/run.py]: ping @chrisfilo 

What about:
```Python
    layout = BIDSLayout(bids_dir, validate=not opts.skip_bids_validator)
```

File: fmriprep/workflows/fieldmap/phdiff.py (hunk 2 above)
Comment (oesteban) [reply fmriprep/workflows/fieldmap/phdiff.py]: Let's submit it, I feel like timing the first creation is somewhat splitting hairs.

File: fmriprep/cli/run.py (hunk 1 above)
Comment (chrisfilo) [reply fmriprep/cli/run.py]: I think `validate=False` is the way to go. `bids-validator` would provide better validation and if it's turned off we should not validate anyway.

Comment (oesteban) [reply fmriprep/cli/run.py]: Oh, okay

**Pull request merged**